config/config/wc-config.yaml

```diff
@@ -105,41 +193,96 @@ elasticsearch:
   clientNode:
     count: 1
 
+## Configuration for fluentd
+##
 fluentd:
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
   tolerations:
   - effect: NoSchedule
     key: "node-role.kubernetes.io/master"
     value: ""
-  # Only run on control plane nodes
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
   nodeSelector:
     node-role.kubernetes.io/master: ""
+
+  ## Define resources requests and limits for single Pods.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources:
     limits:
       cpu: 200m
       memory: 500Mi
     requests:
       cpu: 200m
       memory: 500Mi
+
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  ##
   affinity: {}
+
+  ## Extra fluentd config to mount
+  ##
   extraConfigMaps: {}
+
   user:
```

Comment:
This one I think it would be good to comment on. Something like
> Configuration for the user-controllable fluentd that ships logs from nodes that the user can run pods on.